covalent/_workflow/lattice.py

```diff
@@ -244,8 +244,11 @@ def build_graph(self, *args, **kwargs) -> None:
                     )
                     raise
 
-        if get_config("sdk.exhaustive_postprocess") == "true":
-            pp = Postprocessor(lattice=self)
+        pp = Postprocessor(lattice=self)
+
+        if get_config("sdk.exhaustive_postprocess") != "true":
```

Comment:
This condition seems unnecessarily complicated, can't we just check `sdk.reconstruct_postprocess` is "true" or not?

Reply:
Modified